astroquery/cadc/core.py

```diff
@@ -160,8 +175,8 @@ def login(self, user=None, password=None, certificate_file=None):
                 "Content-type": "application/x-www-form-urlencoded",
                 "Accept": "text/plain"
             }
-            response = pyvo.dal.tap.s.post(login_url, data=args,
-                                           headers=header)
+            response = requests.post(login_url, data=args,
```

Comment:
any reason why not go through `self._request` instead?

Reply:
Forgot about it. Sorry. I'll make the correction.

Reply:
Fixed although I couldn't make use of the bonus functionality (cache) as the call retrieves time sensitive cookies.